looker/data_source_looker_group_test.go

```diff
@@ -0,0 +1,45 @@
+package looker
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccDatasourceLookerGroup(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		ProviderFactories: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				ResourceName: "looker_group",
+				Config: `
+				data "looker_group" "all" {
+					name = "All Users"
+				}
+
+				output "looker_group_all_id" {
+					value = data.looker_group.all.id
+				}
+				`,
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckOutput("looker_group_all_id", "1"),
```

Comment:
Is the ID for `all_users` always going to be 1 or should we manually check that?

Reply:
If it's always 1 then I think this is fine

Reply:
@Vitzeno always seems to be 1. In the UI it shows as a system group that is locked so think this is fine

Reply:
Ahh good point, I think it is because that group is generated by looker 🤞🏾